Assets/MRTK/Services/DiagnosticsSystem/MixedRealityToolkitVisualProfiler.cs

```diff
@@ -437,18 +448,36 @@ private void LateUpdate()
                 }
 
                 // Update visibility state.
-                window.gameObject.SetActive(ShouldShowProfiler);
-                memoryStats.gameObject.SetActive(memoryStatsVisible);
+                if (window.gameObject.activeSelf != ShouldShowProfiler)
+                {
+                    window.gameObject.SetActive(ShouldShowProfiler);
+                }
+
+                if (memoryStats.gameObject.activeSelf != memoryStatsVisible)
+                {
+                    memoryStats.gameObject.SetActive(memoryStatsVisible);
+                }
             }
         }
 
+#if WINDOWS_UWP
+        private void AppCapture_CapturingChanged(AppCapture sender, object args) => appCaptureIsCapturingVideo = sender.IsCapturingVideo;
+        private float previousFieldOfView = -1.0f;
+#endif // WINDOWS_UWP
+
         private static readonly ProfilerMarker CalculateWindowPositionPerfMarker = new ProfilerMarker("[MRTK] MixedRealityToolkitVisualProfiler.CalculateWindowPosition");
 
         private Vector3 CalculateWindowPosition(Transform cameraTransform)
         {
             using (CalculateWindowPositionPerfMarker.Auto())
             {
-                float windowDistance = Mathf.Max(16.0f / CameraCache.Main.fieldOfView, CameraCache.Main.nearClipPlane + 0.25f);
+                float windowDistance =
+#if WINDOWS_UWP
+                    Mathf.Max(16.0f / (appCaptureIsCapturingVideo ? previousFieldOfView : previousFieldOfView = CameraCache.Main.fieldOfView), Mathf.Max(CameraCache.Main.nearClipPlane, 0.5f));
```

Comment:
Ahh you're right...I assume that's possible. I'll take a look.

Reply:
Tested this case out, and since we take a `Max` here, it's okay! `16 / -1` will always be less than the other side, so we default to `Mathf.Max(CameraCache.Main.nearClipPlane, 0.5f)`